forum/models/contents.py

```diff
@@ -1,7 +1,7 @@
 """Content Class for mongo backend."""
 
 from datetime import datetime
-from typing import Any, Dict, List
+from typing import Any, List
```

Comment:
Update List to list

Reply:
LGTM!

Reply:
since `list` (the method) is already defined in the `BaseContents` class, mypy considers it as function instead of list object without the typing import `List`. That's why I kept it explicit.

Alternatively, we can rename the `list` method to `get_all` to avoid this issue.